GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/Content/button.config

```diff
@@ -572,354 +483,6 @@
-            "columnSize": "",
-            "columnSizeFromDesktop": "",
-            "cssClassesForColumn": ""
-          },
-          {
-            "contentTypeKey": "e683e885-8aa0-4bdd-9eff-56966f353540",
-            "udi": "umb://element/bde95f67b02e4cb3a51c9503c9ed9647",
-            "preventDoubleClick": "0",
-            "modelProperty": "",
-            "typeOfButton": "Primary",
-            "styleOfButton": [
-              "Warning"
-            ],
-            "disabled": "0",
-            "cssClasses": "tpr-button--no-next-step",
-            "cssClassesForRow": "",
-            "columnSize": "",
-            "columnSizeFromDesktop": "",
-            "cssClassesForColumn": ""
-          },
-          {
-            "contentTypeKey": "e683e885-8aa0-4bdd-9eff-56966f353540",
-            "udi": "umb://element/ef79a6a569b74bb985861aae00c944d4",
-            "preventDoubleClick": "0",
-            "modelProperty": "",
-            "typeOfButton": "Primary",
-            "styleOfButton": [
-              "Secondary",
-              "Warning"
-            ],
-            "disabled": "0",
-            "cssClasses": "tpr-button--no-next-step",
-            "cssClassesForRow": "",
-            "columnSize": "",
-            "columnSizeFromDesktop": "",
-            "cssClassesForColumn": ""
-          },
-          {
-            "contentTypeKey": "e683e885-8aa0-4bdd-9eff-56966f353540",
-            "udi": "umb://element/96c36f62a5d14e5c831157912d540143",
-            "preventDoubleClick": "0",
-            "modelProperty": "",
-            "typeOfButton": "Primary",
-            "styleOfButton": "",
-            "disabled": "1",
-            "cssClasses": "tpr-button--no-next-step",
-            "cssClassesForRow": "",
-            "columnSize": "",
-            "columnSizeFromDesktop": "",
-            "cssClassesForColumn": ""
-          },
-          {
-            "contentTypeKey": "e683e885-8aa0-4bdd-9eff-56966f353540",
-            "udi": "umb://element/75b4e7129a564728b3727f8b8bfca146",
-            "preventDoubleClick": "0",
-            "modelProperty": "",
-            "typeOfButton": "Primary",
-            "styleOfButton": [
-              "Secondary"
-            ],
-            "disabled": "1",
-            "cssClasses": "tpr-button--no-next-step",
-            "cssClassesForRow": "",
-            "columnSize": "",
-            "columnSizeFromDesktop": "",
-            "cssClassesForColumn": ""
-          },
-          {
-            "contentTypeKey": "e683e885-8aa0-4bdd-9eff-56966f353540",
-            "udi": "umb://element/285b567ae042487cbf7eeee624a9a69c",
-            "preventDoubleClick": "0",
-            "modelProperty": "",
-            "typeOfButton": "Primary",
-            "styleOfButton": [
-              "Warning"
-            ],
-            "disabled": "1",
-            "cssClasses": "tpr-button--no-next-step",
-            "cssClassesForRow": "",
-            "columnSize": "",
```

Comment:
There's a lot of content being removed here which are nothing to do with your change. This indicates you have an outdated version of the 'Buttons' page.

You can see similar problems are happening with quite a few pages.